### ISSUE TITLE
Discovery/Engineering: Data source map for VA.gov, VAHB, & BTSSS

### ISSUE THREAD

**Issue by liztownd** [department-of-veterans-affairs/va.gov-team#91607]
## Description
BTSSS data appears to be siloed away from the rest of the data sources used by VA.gov and VAHB Mobile app.

## User Story
As a developer, I need to understand the system diagrams with downstream/upstream services and data sources for VA.gov, the VAHB Mobile app, and BTSSS, so I can understand where they differ.

## Acceptance Criteria
- [x] System diagrams with data sources for VA.gov, VAHB app, and BTSSS created/located
- [x] Differences in sources/services identified with recommendations for where they can be consolidated

## Definition of Done
- [ ] All acceptance criteria are met
- [ ] Documentation is updated

---
## How to configure this issue
- [x] **Attached to an Epic**

**Comment by kjduensing** [comment]
@liztownd, here's the overarching API confluence space: https://confluence.devops.va.gov/display/VAExternal/Migrations+Team+-+API+Technical+Specs+and+Design+Documentation

The diagram I was referring to yesterday is included in their user guide: https://docs.google.com/document/d/1M3bJpUd_tqBJSdXKMszRvLjoeBe-_QH7/edit#heading=h.ihv636

**Comment by liztownd** [comment]
The main difference will be around Direct Deposit. For `vets-api` this data lives in CorpDB, for Travel Pay it lives in FSC/FMS. 

The other data points required for submission come from the same data stores, and should should be transferrable between systems if updated via a different entry point.

**Comment by kjduensing** [comment]
@liztownd I see this is marked closed - can you link the diagrams you created to this ticket please?

**Comment by liztownd** [comment]
> @liztownd I see this is marked closed - can you link the diagrams you created to this ticket please?

Yeah @kjduensing - I wasn't sure how sensitive the info was to be posting it here. I didn't actually create any diagrams, just found links to various things and asked a lot of questions. 😁 

For VAHB, I referenced this [services map](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/Engineering/BackEnd/Architecture/Services/#upstream-service-map) (although EVSS has been (mostly? completely?) deprecated)

Payment Information data now comes through the Lighthouse Direct Deposit API -> BGS -> CorpDB for `vets-api` (vs. FSC for BTSSS).

For Address and Contact information, I referenced the [VAProfile Data Diagram](https://dvagov.sharepoint.com/sites/OITEPMOVAPROPUB/SitePages/Profile-Service.aspx) (only available on the VA Network) and spoke with the VAProfile PM to verify that the data flows are bi-directional between VAProfile <> ES (which is where BTSSS pulls the data from).

Appointment data comes from VAOS, which pulls from VistA, Oracle, and EHRM/FHIR (?), which are the same data sources for Appointments for BTSSS. BTSSS confirmed that they will be providing the original source Appt. ID in this [DSVA Slack thread](https://dsva.slack.com/archives/C05UTPZRZFY/p1724873485995459).

I believe those are the salient data points required for SMOC at this time, but correct me if I'm wrong and we need to locate source of truth for anything else.
